tests/test_processing_common.py

```diff
@@ -233,13 +236,14 @@ def test_unstructured_kwargs_batched(self):
             images=image_input,
             return_tensors="pt",
             size={"height": 214, "width": 214},
+            crop_size={"height": 214, "width": 214},
```

Comment:
@yonigozlan i think you removed `crop_size` from common tests and it had smth to do with some image processors accepting/not accepting certain kwargs?

Reply:
@zucchini-nlp Yes but actually it would be nice to have both here. @molbap had some CI tests crash because `crop_size` was removed here and the image_processor had `do_center_crop` set to True by default which canceled out size. Having both would handle cases where either `do_center_crop` is set to True in the image_processor by default, or `crop_size` is not supported by the image_processor.
So I am for keeping this and merging this PR before some other kwargs uniformization PRs